docs/conf.py

```diff
@@ -29,7 +30,9 @@
 import pykiso
 
 project = "pykiso"
-copyright = "2020, Sebastian Fischer, Daniel Bühler, Damien Kayser"
+copyright = (
+    f"{datetime.date.today().year}, Sebastian Fischer, Daniel Bühler, Damien Kayser"
```

Comment:
We should add all our names here for the ones whos wants to be listed.. 
You could add my Name (:

Reply:
Will do ;)